README.md

```diff
@@ -44,18 +44,19 @@ Log in to Keycloak using the credentials stored in the `credential-rhsso-backsta
 ### Backstage
 Configure Backstage and deploy to OpenShift.  The following commands must be run from within the [assemble-backstage](./charts/assemble-backstage) directory.
 
-#### OAuth Configuration
-Update the `oauth` section of `values.yaml` to the following:
+#### RHSSO Configuration
+Update the `rhsso` section of `values.yaml` to the following:
 ```
-oauth:
-  enabled: true
+rhsso:
+  # Generate this through the command line using
+  # echo "https://keycloak-backstage.apps$(oc cluster-info | grep -Eo '.cluster(.*?).com')/auth"
+  issuerUrl: https://<RHSSO_HOST>/auth
```

Comment:
In the values.yaml this is `baseUrl`
Also in the values.yaml has a `realm` that is missing here

Reply:
Updated